docs/openapi/components/schemas/credentials/DeliveryScheduleCredential.yml

```diff
@@ -0,0 +1,189 @@
+      - https://w3id.org/traceability/v1
+    items:
+      type: string
+      enum:
+        - https://www.w3.org/2018/credentials/v1
+        - https://w3id.org/traceability/v1
+  type:
+    type: array
+    readOnly: true
+    const:
+      - VerifiableCredential
+      - DeliveryScheduleCredential
+    default:
+      - VerifiableCredential
+      - DeliveryScheduleCredential
+    items:
+      type: string
+      enum:
+        - VerifiableCredential
+        - DeliveryScheduleCredential
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    $ref: ../common/Organization.yml
+  credentialSchema:
+    type: object
+    properties:
+      id:
+        title: Id
+        description: The URL of the schema file to use to validate the shape of the JSON object
+        type: string
+        format: uri
+        example: https://w3id.org/traceability/openapi/components/schemas/credentials/DeliveryScheduleCredential.yml
+        default: https://w3id.org/traceability/openapi/components/schemas/credentials/DeliveryScheduleCredential.yml
+        readonly: true
+      type:
+        title: Type
+        description: The type of validation to be run against the defined schema
+        const: OpenApiSpecificationValidator2022
+  credentialSubject:
+    $ref: ../common/DeliverySchedule.yml
+  proof:
+    type: object
+additionalProperties: false
+required:
+  - '@context'
+  - type
+  - issuanceDate
+  - issuer
+  - credentialSubject
+example: |-
+  {
+    "type": [
+      "VerifiableCredential",
+      "DeliveryScheduleCredential"
+    ],
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "did:key:z6LSpdSReUHCjYcQb1243aF1vS7sd9ArK585Mm4ktARQxatd",
+    "issuer": {
+      "type": [
+        "Organization"
+      ],
+      "id": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+      "name": "Everything Garden Stuff"
+    },
+    "issuanceDate": "2022-01-13T09:16:46Z",
+    "credentialSubject": {
+      "type": [
+        "DeliverySchedule"
+      ],
+      "transporter": {
+        "type": [
+          "Organization"
+        ],
+        "name": "Example Org"
+      },
+      "batchNumber": "111-1111-11",
+      "commodity": {
+        "type": [
+          "Commodity"
+        ],
+        "commodityCode": "2714.10.00.00",
+        "commodityCodeType": "HTS"
+      },
+      "receiptLocation": {
+        "type": [
+          "Place"
+        ],
+        "geo": {
+          "type": [
+            "GeoCoordinates"
+          ],
+          "latitude": "52.6735",
+          "longitude": "-111.3075"
+        },
+        "address": {
+          "type": [
+            "PostalAddress"
+          ],
+          "streetAddress": "",
+          "addressLocality": "Hardisty",
+          "addressRegion": "AB",
+          "postalCode": "",
+          "addressCountry": "CAN"
```

Comment:
We are recommending ourselves to use 2-char ISO country codes. Let's align to our own recommendation.